### ISSUE TITLE
Bereitstellung eines Download Pakets als Electron App

### ISSUE THREAD

**Issue by HerrLoesch** [zeiss-digital-innovation/DecisionMatrixHelper#4]
Ich möchte das Tool auch offline nutzen können. Das möchte ich mit Electron umgesetzt sehen. In diesem GitHub Repo soll daher in der Readme ein Download Button für den letzten stabilen Release bereitgestellt werden.

**Comment by eshadepunkt** [comment]
Das Tool ist bereits als PWA verfügbar und damit auch offline nutzbar (inklusive cache) - soll es dennoch als electron app umgesetzt werden?

**Comment by HerrLoesch** [comment]
Ja. Bitte einen Buildtask per GitHub Actions machen. Wenn auf dem Main eine neue Version liegt soll automatisch eine downloadbare Version in Electron erstellt werden. Die man über die Readme des GitHub Repos runter laden kann.

**Comment by eshadepunkt** [comment]
![image](https://user-images.githubusercontent.com/72971415/221153940-110d3e97-be51-4d43-8c67-f6d10839a610.png)
Aktuell wird es über den Button des Browsers oben rechts in der Adressleiste gelöst. 

TODO: Event abfangen und als Button in der Page bereitstellen
